apps/framework-cli/src/utilities/package_managers.rs

```diff
@@ -40,6 +40,71 @@ pub enum PackageManager {
     Pnpm,
 }
 
+/// Gets the installed pnpm version by running `pnpm --version`.
+///
+/// # Returns
+///
+/// * `Option<String>` - The version string (e.g., "10.24.0") or None if pnpm is not installed
+pub fn get_pnpm_version() -> Option<String> {
```

Comment:
I wonder if we should have a proper `Version` object to manipulate versions instead of a String.
That's more in general and not blocking